index.js

```diff
@@ -0,0 +1,15 @@
+var express = require( "express" );
+var app;
+var server;
+var port;
+
+//jscs:disable requireCamelCaseOrUpperCaseIdentifiers
+port = process.env.npm_package_config_port;
+
+//jscs:enable requireCamelCaseOrUpperCaseIdentifiers
+
+app = express();
+app.use ( "/", express.static( "./static" ) );
+server = app.listen( port, function() {
+	console.log( "Listening on http://localhost:" + port );
+} );
```

Comment:
Do we want to log all incoming requests onto console ? I'll work later on logging to a file.